site/src/components/pages/blocks/block-list-container.tsx

```diff
@@ -0,0 +1,23 @@
+import { experimental_sx as sx, styled } from "@mui/material";
+
+import { BlockFormContainer } from "./block-form-container";
+
+export const BlockListContainer = styled(BlockFormContainer)<{
+  hasBlocks?: boolean;
+}>(
```

Comment:
nit: I don't think we need to create a new file for component style overrides like this that have few lines and are used in one place. I feel it'll quickly become difficult to find components that cause certain styles since one would have to jump through files just to update a particular style.   I'd prefer to place them in the same file they are used if the styles are small like this or better still have a single `block-form-styles` file that exports all the style-based components needed in block form. 

I think it's better we create a new file for styled components when 
1) It has a lot of styles 
2) It is reused in more than one component